typings.d.ts

```diff
@@ -108,6 +108,6 @@ declare module '@redux-offline/redux-offline' {
       preloadedState: T,
       enhancer: StoreEnhancer<T>,
     ) => Store<T>,
-    middleware: <A = any, R = any>(store: any) => (next: (action: A) => R) => (action: A) => R,
+    middleware: (api: MiddlewareAPI<any>) => (next: Dispatch<any>) => Dispatch<any>,
```

Comment:
I didn't know about MiddlewareAPI nice! Just one thing, you made me look into it and there's another type called Middleware. Wouldn't this suffice?

Reply:
This is enough for now, because we still support redux v3.